python/hopsworks/git_commit.py

```diff
@@ -17,7 +17,7 @@
 import json
 
 import humps
-from hopsworks import util
+from hopsworks_common import util
 
 
 class GitCommit:
```

Comment:
This file looks wrong, should it not be importing `git_commit `from `hopsworks_common`?

Reply:
Yes, for sure. Thanks for catching it!